include/xwayland/selection.h

```diff
@@ -20,35 +20,45 @@ struct wlr_xwm_selection_transfer {
 	struct wl_array source_data;
 	int wl_client_fd;
 	struct wl_event_source *event_source;
+	struct wl_list link;
 
 	// when sending to x11
 	xcb_selection_request_event_t request;
-	struct wl_list outgoing_link;
 
 	// when receiving from x11
 	int property_start;
 	xcb_get_property_reply_t *property_reply;
+	xcb_window_t window;
```

Comment:
Maybe we should consider renaming this to `incoming_window`, to make it clear it's unused for outgoing.

Reply:
Sure, done.